common/gpu/intel/opencl.nix

```
@@ -0,0 +1,14 @@
+{ config, lib, pkgs, ... }:
+
+{
+  options.hardware.intel.opencl.enable = lib.mkEnableOption (lib.mkDoc
+    "intel opencl runtime (Install icd for Intel GPUs)"
+  );
```

No strong opinion here...:

```suggestion
  options.hardware.intel.opencl.enable = lib.mkEnableOption (lib.mkDoc
    "intel opencl runtime (Install icd for Intel GPUs)"
  ) // {
    enable = true;
  };
```